draft-ietf-pquip-pqc-engineers.md

```diff
@@ -213,7 +213,7 @@ The editors actively encourage contributions to this document. Please consider w
 
 Any asymmetric cryptographic algorithm based on integer factorization, finite field discrete logarithms or elliptic curve discrete logarithms will be vulnerable to attacks using Shor's Algorithm on a sufficiently large general-purpose quantum computer, known as a CRQC. This document focuses on the principal functions of asymmetric cryptography:
 
-* Key Agreement and Key Transport:  Key Agreement schemes, typically referred to as Diffie-Hellman or Elliptic Curve Diffie-Hellman, as well as Key Transport, typically using RSA Encryption, are used to establish a shared cryptographic key for secure communication. They are one of the mechanisms that can be replaced by PQC, as this is based on public key cryptography and is therefore vulnerable to the Shor's algorithm. An CRQC can employ Shor's algorithm to efficiently find the prime factors of a large public key, which in turn can be exploited to derive the private key.
+* Key Agreement and Key Transport:  Key Agreement schemes, typically referred to as Diffie-Hellman (DH) or Elliptic Curve Diffie-Hellman (ECDH), as well as Key Transport, typically using RSA Encryption, are used to establish a shared cryptographic key for secure communication. They are one of the mechanisms that can be replaced by PQC, as this is based on public key cryptography and is therefore vulnerable to the Shor's algorithm. An CRQC can employ Shor's algorithm to efficiently find the prime factors of a large public key (in case of RSA), which in turn can be exploited to derive the private key. Additionally, for Diffie-Hellman, a CRQC can find the exponent or discrete logarithm of the (short or long term) DH public key, which is the exact secret to derive the session key.”
 
```

Comment:
I would re-phrase the line as follows:
In the case of Diffie-Hellman, a CRQC has the potential to calculate the exponent or discrete logarithm of the (short or long-term) Diffie-Hellman public key. This, in turn, would reveal the precise secret required to derive the session key.